resources/views/auth/anauthorized.blade.php

```diff
@@ -0,0 +1,2 @@
+<h1>You Don't have the proper priviledges to do this!</h1>
```

Comment:
OH! One little thing. This file has a typo in the name: `anauthorized.blade.php`. 👾